### ISSUE TITLE
What's the purpose of bluetooth-rx if we can't decrypt Bluetooth Classic packets?

### ISSUE THREAD

**Issue by SamDecrock** [greatscottgadgets/ubertooth#312]
Just a quick question,

I initially bought the Ubertooth One to reverse engineer the protocol of Amazon's Echo Buttons. They use regular Bluetooth Classic.

After discovering the UAP, I'm able to capture and visualize the Bluetooth Classic packets using

```
mkfifo /tmp/pipe
sudo wireshark -k -i /tmp/pipe
ubertooth-rx -l <LAP> -u <UAP> -q /tmp/pipe
```

After a while I see a packet payloads in the ubertooth-rx console, but in my Wireshark window it says `encrypted data`.

I have seen many replies by @dominicgs that decrypting Bluetooth Classic packets is not possible.

But... what's the purpose of the bluetooth-rx tool then?

(don't want to be rude here, I love the work you have put into Ubertooth ... it's great for decrypting BLE packets. I just wished there was a way to decrypt BT Classic packets)

**Comment by dominicgs** [comment]
`ubertooth-rx` is most frequently used for finding the LAP and UAP of non-discoverable devices.

In certain circumstances, which were more common when we first wrote the tool, BT Classic devices operate unencrypted and `ubertooth-rx` was able to follow them for a time.  This just isn't as common these days, and even if we were able to capture all BT classic channels simultaneously, we could still only crack the encryption on BT 1.x devices, because 2.x use a better key exchange protocol (that I believe has not been broken yet).
